server/validators/dateIsSameOrBefore.ts

```diff
@@ -6,7 +6,7 @@ import SimpleDate from '../commonValidationTypes/simpleDate'
 export default function DateIsSameOrBefore(dateToCompare: (o: any) => Date, validationOptions?: ValidationOptions) {
   const dateIsSameOrBefore = (date: SimpleDate, args: ValidationArguments) =>
     isValid(date.toRichDate())
-      ? dateToCompare(args.object) === null || date.toRichDate() <= startOfDay(new Date(dateToCompare(args.object)))
+      ? dateToCompare(args.object) == null || date.toRichDate() <= startOfDay(new Date(dateToCompare(args.object)))
```

Comment:
@smcveigh941 is this right? The rational being the returned value could be null or undefined. For example there might be an activity end.

If so not sure how to test it.